src/main/java/pl/jkap/sozzt/contract/ContractController.java

```diff
@@ -0,0 +1,49 @@
+package pl.jkap.sozzt.contract;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import pl.jkap.sozzt.contract.domain.ContractFacade;
+import pl.jkap.sozzt.contract.dto.ContractDto;
+
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+@SuppressWarnings(value = "unused")
+public class ContractController {
+
+    @Autowired
```

Comment:
@Autowired on field is not a recommended way of injecting beanse, `@RequiredArgsConstructor` is doing everything you need to - you can read why field injection is worse then constructor, one small hint from me is to think about it from the perspective of tests and what is easier to test